frontend/src/components/organisms/form-sections/OrganisationSection/TechnicalTeamCard.js

```diff
@@ -44,33 +85,64 @@ export const TechnicalTeamCard = ({ editorList = [] }) => {
 
   return (
     <Card>
-      <h3>Pour implémenter l‘API</h3>
-      <SelectInput
-        label="Qui va implémenter l’API ?"
-        options={[
-          { id: '', label: 'Sélectionner une option' },
-          { id: 'internal_team', label: 'Développeurs en interne' },
-          { id: 'software_company', label: 'Éditeur de logiciel' },
-          { id: 'other', label: 'Autre' },
-        ]}
-        name="technical_team_type"
-        value={technical_team_type}
-        disabled={disabled}
-        onChange={onTypeChange}
-        required
-      />
-      {technical_team_type === 'software_company' && (
-        <TextInputWithSuggestions
-          label="Quel est votre éditeur de logiciel ?"
-          options={editorOptions}
-          name="technical_team_value"
-          value={technicalTeamValueLabel}
-          disabled={disabled}
-          onChange={onValueChange}
-          required
-        />
+      <h3>Qui implémentera l’API ?</h3>
+      {!isUserEnrollmentLoading && !technical_team_type && (
```

Comment:
There is mutually exclusive contents depending on isUserEnrollmentLoading and technical_team_type
The code do not reflect intents and the conditions are a bit hard to follow.
Also in general keep this kind of logic out of the jsx
I provide here an example using smaller logic blocks but cleaner implementation with other component(s) may be better.

```tsx

  const technicalTeamTypeInput = !!technical_team_type ? (
    <>
      {disabled ? null : (
        <div
          style={{
            marginBottom: '0.5rem',
          }}
        >
          <Button onClick={resetType} icon="arrow-left" outline>
            retour
          </Button>
        </div>
      )}
      <RadioInput
        label="Qui s’occupera des aspects techniques et informatiques ?"
        options={[{ id: '', label: selectedTypeLabel }]}
        name="technical_team_type"
        value=""
        disabled={disabled}
        onChange={() => null}
        required
      />
    </>
  ) : (
    <>
      <Alert>
        Pour lire les données vous aurez besoin d’un logiciel ou d’un service en
        ligne. <br />
        <Link href="/faq#champ-editeur" inline newTab>
          En savoir plus
        </Link>
      </Alert>
      <RadioInput
        label="Qui s’occupera des aspects techniques et informatiques ?"
        options={typeOptions}
        name="technical_team_type"
        value={technical_team_type}
        disabled={disabled}
        onChange={onTypeChange}
        required
      />
    </>
  );

  // Display relevant technicalTeamValue input depending on teamType
  const technicalTeamValueInput =
    technical_team_type === 'software_company' ? (
      <TextInputWithSuggestions
        label="Quel est votre éditeur de logiciel ?"
        options={editorOptions}
        name="technical_team_value"
        value={technicalTeamValueLabel}
        disabled={disabled}
        onChange={onValueChange}
        required
      />
    ) : technical_team_type === 'other' ? (
      <TextInput
        label="Précisez ?"
        name="technical_team_value"
        value={technical_team_value}
        disabled={disabled}
        onChange={onChange}
        required
      />
    ) : null;

  const cardContent = isUserEnrollmentLoading ? null : (
    <>
      {technicalTeamTypeInput}
      {technicalTeamValueInput}
    </>
  );

  return (
    <Card>
      <h3>Qui implémentera l’API ?</h3>
      {cardContent}
    </Card>
  );
  ```